docs/content/Configuration/Config.md

```diff
@@ -70,6 +70,7 @@ interface CubejsConfiguration {
   },
   externalDbType: string | ((context: RequestContext) => string);
   externalDriverFactory: (context: RequestContext) => BaseDriver | Promise<BaseDriver>;
+  cacheAndQueueDriver: string;
```

Comment:
I think it's better to write `memory | redis`

Reply:
Also `ioredis`?